src/test/java/com/ait/lienzo/client/core/shape/toolbox/ToolboxVisibilityExecutorsTest.java

```diff
@@ -0,0 +1,17 @@
+package com.ait.lienzo.client.core.shape.toolbox;
+
+import com.ait.lienzo.test.LienzoMockitoTestRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class ToolboxVisibilityExecutorsTest {
+
+    @Test
+    public void animationDelay() {
```

Comment:
Test method name should have `test` prefix.

Reply:
@hasys, Sure! It is already fixed.